packages/es-components/webpack-components-bundle.config.js

```diff
@@ -2,7 +2,6 @@ const path = require('path');
 
 module.exports = {
   entry: './src/index.js',
-  mode: 'production',
```

Comment:
But why?

Reply:
This is because we have to stay on Webpack 3 until react-styleguidist updates to Webpack 4. You can see discussion around that [in their issues](https://github.com/styleguidist/react-styleguidist/issues/915).

Reply:
I fail to the the relevance of that issue. Webpack is a dependency for styleguidist, not a peer dependency. That means the version of webpack we're specifying has not bearing on styleguidist. I also fail to see how our being on webpack 4 is impacting both our ability to run the style guide locally and how it functions deployed to github pages.

Reply:
`react-styleguidist` has `webpack-dev-middleware` and `webpack-dev-server` as dependencies, which each require peer a peer dependency of webpack 3 or less.

This is the exact output from npm when I upgrade to webpack 4:
> npm WARN webpack-dev-middleware@1.12.2 requires a peer of webpack@^1.0.0 || ^2.0.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.
> npm WARN webpack-dev-server@2.11.3 requires a peer of webpack@^2.2.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.

Everything seems to run fine, so if we're ok with those warnings, I can just update it.

Reply:
I don't think this is an issue at all. That warning stems from running node v6/npm v3. Travis is configured to run against the latest node and LTS releases(v10 and v8, respectively), and I have yet to see a warning show up in the travis build logs concerning peer dependencies. I don't see any particular reason to sacrifice bundle optimization and dropping down a version to address dev time console warnings from tooling with end-of-life dated for about 6-7 months from now.